lib/lambda_handler.rb

```diff
@@ -0,0 +1,22 @@
+require 'nasta/reports'
+
+def lambda_handler(event:, context:)
+  report_name = event['report']
+
+  return { statusCode: 400, body: 'No report was specified' } if report_name.nil?
+
+  begin
+    class_name = Object.const_get(report_name)
+  rescue NameError => e
+    return { statusCode: 404, body: 'Report #{report_name} not found' }
+  end
+
+  begin
+    report = class_name.new
+    report.fetch
```

Comment:
čo má robiť fetch? Myslel som že len fetchne dáta, a zápis prebehne následne. Či?

Reply:
@samuelszabo ak vieme mat univerzalny zapis, tak to mozeme zvazit, ale to ja teraz neviem posudit

Reply:
Poďme do toho. Takto budeme zbytočne mať duplicitné kódy a momentálne nevieme o inom zápise ako time+value.